schemas/veres-one-validator.js

```diff
@@ -5,12 +5,19 @@
 
 const {config: {constants}} = require('bedrock');
 const {schemas} = require('bedrock-validation');
+const {maxLength} = require('../lib/constants');
 const did = require('./did');
 const didReference = require('./didReference');
 const didUuid = require('./did-uuid');
 const {serviceDescriptor, serviceId} = require('./service');
 const urnUuid = require('./urn-uuid');
 
+// can be a did or a url
+const creator = {
+  // FIXME this will allow any string with minLength 1 or more
+  anyOf: [schemas.url(), did(), didReference(), didUuid()]
```

Comment:
When is creator ever a DID uuid or a `schemas.url`? This seems too restrictive. Can we minimize this?

Reply:
It looks like this is due to a larger issue. `creator` is always a URL in all test data up to this point.

Reply:
I'll add a punchlist item for ensuring veres-one validators only accept specific URL types.